evento/src/components/TabContainer/index.js

```diff
@@ -6,18 +6,34 @@ const Tab = (props) => {
 	return (
 		<div className="Tab">
 			<h4>
-				<Link to={props.path}> {props.title} </Link>
+				<Link to={props.path} onClick={() => props.onClick(props.id)}> {props.title} </Link>
 			</h4>
 		</div>
 	);
 };
 
 class TabContainer extends Component {
+	constructor() {
+		super();
+		this.state = { selectedId: 1 };
```

Comment:
Current url can be accessed with browserHistory.location (should be in this.props.history.location as well) and we can just match the current url against all tab.path's.